components/grid/columns/checkbox.md

```diff
@@ -27,6 +27,7 @@ The Grid checkbox column has the following exclusive parameters. For other avail
 | `CheckBoxOnlySelection` | `bool` | Determines if row selection occurs only on checkbox clicks. By default, user can select rows by clicking anywhere, except on command buttons. |
 | `SelectAll` | `bool` <br /> (`true`) | Determines if the column header renders a checkbox to select all rows. Set this to `false` if the [Grid `SelectionMode` is `Single`]({%slug components/grid/selection/single%}). The `SelectAll` parameter has no effect when the checkbox column has a [`HeaderTemplate`](#headertemplate). |
 | `SelectAllMode` | `GridSelectAllMode` enum <br /> (`Current`) | Determines if the header cell checkbox selects all rows on the current page, or all rows in the Grid. `Current` selects the visible rows on the current page. `All` selects all the data items, including ones that may be currently filtered out. `All` requires the [Grid to be data-bound via its `Data` parameter, and not `OnRead`]({%slug common-features-data-binding-overview%}#how-to-provide-data). When using `OnRead`, the two `SelectAllMode`s behave identically, because the Grid controls only one page of items. |
+| `Title` | `string` | The text in the checkbox column's header. The title renders only when `SelectAll` is `false`. |
```

Comment:
I'd expect that if `SelectAll="false"` and we have a `HeaderTemplate`, the `Title` is not rendered. However, this is not the current behavior.

 If `SelectAll="false"` the `HeaderTemplate` content is not rendered. Instead, the `Title` is shown if provided. Is this by design? If yes, we should document it as well. If not, we should open a bug.

Reply:
Bug fixed in the source.